cmd/build.go

```diff
@@ -27,33 +26,35 @@ import (
 var dotSlash = string([]byte{'.', filepath.Separator})
 
 var (
-	buildTarget            string
-	buildBranch            string
-	buildDebug             bool
-	buildCachePath         string
-	buildOmitEmbedder      bool
```

Comment:
Why is skipping the embedder compiling removed?

Reply:
I think because it wasn't used in the build subcommand?